README.md

```diff
@@ -1,11 +1,18 @@
 # PyCon Uganda
 
```

Comment:
@rukundob451 thank you for taking off time to make this contribution.

i do have a few takes on this;

1. we could add a description of what pycon is as the first section, perharps under a leading section called `About PyconUg` . Find the information about pycon [here](https://github.com/PyconUg/ug.pycon.org/blob/0cfeccde04727883f36cff3895002c3a8238eaf2/components/AboutSection.js#LL10C9-L10C9)